src/consensus/raft/raft.h

```diff
@@ -28,7 +28,7 @@ namespace raft
     std::vector<Index> terms;
 
   public:
-    static constexpr Term InvalidTerm = 0;
+    static constexpr Term InvalidTerm = ccf::VIEW_UNKNOWN;
```

Comment:
As discussed in #1533, can we use NoVersion everywhere?

Reply:
Let's discuss this on #1533